### PR TITLE
#14 added router component

### PR DESCRIPTION
**Pull request by RossellaFer**
- Added `Router` component using `react-dom-router` 
- Created `Resources` component with minimum JSX
- Routes include / for `App` component and /resources for `Resources` component
- added href for /resources to header

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const Header = () => {
                         </a>
                     </li>
                     <li className="option mobileOption" onClick={closeMobileMenu}>
-                        <a href="#">
+                        <a href="/resources">
```
Comment (sunithapatel) [inline src/components/Header/Header.js:30]: Is it better to change these to `Link` from React Router? 
https://reactrouter.com/web/api/Link

From what I can tell, keeping the `a` tags here will cause a page refresh whereas the React Router `Link` will not. Page refresh would become an issue if we use incorporate usage of state later.

File: src/components/Resources/Resources.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../../assets/css/App.css';
+import './Resources.css';
+import Header from '../Header/Header';
+
+export default function Resources() {
+  return (
+    <>
+      <Header />
```
Comment (sunithapatel) [inline src/components/Resources/Resources.js:9]: Question here: Should the Header component be outside the individual pages? So in App we can have Header once and then Router and router would route to each individual page. 
Does that make sense?

Comment (RossellaFer) [reply src/components/Resources/Resources.js]: I tried to follow this structure
```
index.js
  - App.js
    - Header
    - Router
```
However I got an error message saying that `Link` should be enclosed within the `Router`. In the PR, I moved the `Header` component inside the `Router`, so it does not get refreshed for each route

**Comment by princiya** [comment]
@sunithapatel I have approved this PR. let me know if you have further comments or feedback, if not we can merge this one.

**Comment by princiya** [comment]
@RossellaFer there is a merge conflict on Header.js, we merged another PR just now. Sorry about that, please resolve the conflicts when you get time, if you need help with resolving merge conflicts let us know, we can help you. :)

**Comment by sunithapatel** [comment]
@RossellaFer Sorry about this but when I set up GitHub actions to build and deploy the site, it treats warnings as errors. I had to change the urls in Header.js to resolve this.
Now your changes will conflict with this.
Can you choose all your changes when resolving the conflict and keep the /about and /contact for the other links? Otherwise the build will fail (I think we should pay attention to warnings and remove them anyway).

Please let me know if I can help you with this!

**Comment by sunithapatel** [comment]
Yay thank you @RossellaFer! Awesome job!!!
 I approved it so once it is done with the build I will merge it and hopefully if the GitHub Action works and it will deploy the changes to the site.

**Pull request merged**